core/async/src/messaging.rs

```diff
@@ -7,7 +7,16 @@ use std::fmt::{Debug, Display};
 use std::sync::Arc;
 use tokio::sync::oneshot;
 
-/// Trait for sending a typed message.
+/// Trait for handling a message.
+/// This works in unison with the [`CanSend`] trait. An actor implements the Handler trait for all
```

Comment:
Perhaps mention that "an actor" is an example? This framework is supposed to be not actix-specific.

Reply:
Right, I was taking Actor to be something generic and actix-actor to be an actix implementation for that. But yes, I'll make it more clear in the documentation.